gui-wm/hyprland/hyprland-0.40.0.ebuild

```diff
@@ -0,0 +1,147 @@
+# Copyright 2023-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson toolchain-funcs
+
+DESCRIPTION="A dynamic tiling Wayland compositor that doesn't sacrifice on its looks"
+HOMEPAGE="https://github.com/hyprwm/Hyprland"
+
+if [[ "${PV}" = *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/hyprwm/${PN^}.git"
+else
+	SRC_URI="https://github.com/hyprwm/${PN^}/releases/download/v${PV}/source-v${PV}.tar.gz -> ${P}.gh.tar.gz"
+	S="${WORKDIR}/${PN}-source"
+
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="BSD"
+SLOT="0"
+IUSE="X legacy-renderer systemd"
+
+# hyprpm (hyprland plugin manager) requires the dependencies at runtime
+# so that it can clone, compile and install plugins.
+HYPRPM_RDEPEND="
+	app-alternatives/ninja
+	dev-build/cmake
+	dev-build/meson
+	dev-vcs/git
+	virtual/pkgconfig
+"
+# bundled wlroots has the following dependency string according to included headers.
+# wlroots[drm,gles2-renderer,libinput,x11-backend?,X?]
+# enable x11-backend with X and vice versa
+WLROOTS_DEPEND="
+	>=dev-libs/wayland-1.22
+	media-libs/libglvnd
+	media-libs/mesa[egl(+),gles2]
+	>=x11-libs/libdrm-2.4.114
+	x11-libs/libxkbcommon
+	>=x11-libs/pixman-0.42.0
+	media-libs/libdisplay-info
+	sys-apps/hwdata
+	>=dev-libs/libinput-1.14.0:=
+	sys-auth/seatd:=
+	virtual/libudev:=
+	X? (
+		x11-libs/libxcb:=
+		x11-libs/xcb-util-renderutil
+		x11-libs/xcb-util-wm
+		x11-base/xwayland
+	)
+"
+WLROOTS_RDEPEND="
+	${WLROOTS_DEPEND}
+"
+WLROOTS_BDEPEND="
+	>=dev-libs/wayland-protocols-1.32
+	dev-util/hyprwayland-scanner
+	virtual/pkgconfig
+"
+RDEPEND="
+	${HYPRPM_RDEPEND}
+	${WLROOTS_RDEPEND}
+	dev-cpp/tomlplusplus
+	dev-libs/glib:2
+	dev-libs/libinput
+	>=dev-libs/wayland-1.20.0
+	gui-libs/hyprcursor
```

Comment:
the hyprland-0.39.1 has a dep of >=gui-libs/hyprcursor-0.1.7, can you double check any changes in this version 0.40.0?

Reply:
Fixed now